tests/IceRpc.Tests/IceProtocolConnectionTests.cs

```diff
@@ -22,7 +22,7 @@ private static IEnumerable<TestCaseData> DispatchFailureSource
                 new InlineDispatcher(
                     (request, cancellationToken) =>
                         new(new OutgoingResponse(request, StatusCode.InvalidData, message: null, invalidDataException))),
-                StatusCode.InternalError,
+                StatusCode.InvalidData,
```

Comment:
We need an extra test for an unmapped StatusCode, for example DeadlineExceeded.

Reply:
we already have DeadlineExceeded below:

```
            yield return new TestCaseData(
                new InlineDispatcher(
                    (request, cancellationToken) =>
                        new(new OutgoingResponse(request, StatusCode.DeadlineExceeded, "deadline message"))),
                StatusCode.InternalError,
                "deadline message { Original StatusCode = DeadlineExceeded }");
```